### ISSUE TITLE
Application distribution not working as noted in wiki

### ISSUE THREAD

**Issue by ngreco** [atom/electron#3374]
Electron v0.34.0
Operating System: Windows 7 x64

__Problem:__
When following the [application distribution tutorial](https://github.com/atom/electron/blob/master/docs/tutorial/application-distribution.md) by creating a folder under electron/resources named app, placing my application files there, and running electron.exe, "nothing happens".

__Expected Result:__
My application should run when I run electron.exe.

__Actual Result:__
If I don't delete the default_app folder and the atom.asar file, the electron just opens like it normally does. If I do delete those items, the electron application either opens and immediately crashes (without telling me why), or just doesn't open. I don't know which. 

__Other Information:__
My application uses a C++ addon that reads data from csv files and manipulates it, returning the data to my application.

**Comment by etiktin** [comment]
Did you copy your package.json to the app folder?
If you did then paste the content of package.json and the file list of app folder.

**Comment by ngreco** [comment]
I did copy package.json to the app folder.
Its contents are as follows:
```
{
  "name"    : "eRedbook",
  "version" : "0.1.0",
  "main"    : "main.js",
}
```
And the file list of the app folder:
```
build/
css/
data/
js/
binding.gyp
changelog.html
data.db
execRedbook.cpp
execRedbook2.cpp
index.html
main.js
package.json
redbookData.cpp
redbookData.h
redbookData.lib
redbookData.obj
sql.js
```

**Comment by etiktin** [comment]
You have an extra comma after the last item in your package.json so it will fail to parse.

**Comment by ngreco** [comment]
Ah, good catch. With that comma gone, the second part of the "actual results" posted above (where the app either doesn't open or closes immediately) occurs.
This application does run as expected when I just drag my main.js onto the electron.exe from an outside directory, so I'm confused as to what's going on.

**Comment by etiktin** [comment]
Run your app from a terminal with the `--enable-logging` flag so you can see if any errors are printed.

You said before that you deleted default_app and atom.asar, did you restore them? atom.asar is mandatory.

**Comment by ngreco** [comment]
Alright, thank you for that flag. Not sure where I would have looked for that.
The problem ended up being that my data folder, where my csv files are being read from, is being looked for under the electron/ folder rather than my app/ folder. One fix is to move my data folder up to the electron folder, another fix is to just fix the path in my code.

I did various attempts with deleting/keeping those files. I did not know that atom.asar was necessary, but now I do. Thanks!
